core/src/main/java/io/grpc/internal/ExponentialBackoffPolicy.java

```diff
@@ -39,26 +39,52 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Retry Policy for Transport reconnection.  Initial parameters from
- * https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md
- *
- * <p>TODO(carl-mastrangelo): add unit tests for this class
+ * Exponential backoff policy for reconnects and call retries.
  */
 final class ExponentialBackoffPolicy implements BackoffPolicy {
-  static final class Provider implements BackoffPolicy.Provider {
+  /**
+   * Provider tuned for connection retries.
+   *
+   * https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md
+   */
+  static final class ReconnectProvider implements BackoffPolicy.Provider {
+    @Override
+    public BackoffPolicy get() {
+      return new ExponentialBackoffPolicy(new Random(), TimeUnit.SECONDS.toMillis(1),
+              TimeUnit.MINUTES.toMillis(2),  1.6, .2);
```

Comment:
No, but there's `TODO` comment in the code which I planned to address right after this PR.